Triangulation_3/include/CGAL/Delaunay_triangulation_3.h

```diff
@@ -919,6 +975,88 @@ class Delaunay_triangulation_3<Gt, Tds_, Default, Lock_data_structure_>
     }
   };
 
+  // Functor for parallel insert_with_info(begin, end) function
+  template <typename DT>
+  class Insert_point_with_info
+  {
+    typedef typename DT::Point                                          Point;
+    typedef typename DT::Triangulation_data_structure::Vertex::Info     Info;
+    typedef typename DT::Vertex_handle                                  Vertex_handle;
+
+    DT                                                  & m_dt;
+    const std::vector<Point>                            & m_points;
+    const std::vector<Info>                             & m_infos;
+    const std::vector<std::ptrdiff_t>                   & m_indices;
+    tbb::enumerable_thread_specific<Vertex_handle>      & m_tls_hint;
+
+  public:
+    // Constructor
+    Insert_point_with_info(DT & dt,
+                 const std::vector<Point> & points,
+                 const std::vector<Info> & infos,
+                 const std::vector<std::ptrdiff_t> & indices,
+                 tbb::enumerable_thread_specific<Vertex_handle> & tls_hint)
+    : m_dt(dt), m_points(points), m_infos(infos), m_indices(indices), m_tls_hint(tls_hint)
+    {}
+
+    // Constructor
```

Comment:
the copy-constructor is needed by [tbb::parallel_for](https://www.threadingbuildingblocks.org/docs/help/reference/algorithms/parallel_for_func.htm)